sys-process/audit/audit-3.0.6.ebuild

```
@@ -144,14 +140,18 @@ multilib_src_install_all() {
 	# newinitd "${FILESDIR}"/auditd-init.d-2.4.3 auditd
 	# newconfd "${FILESDIR}"/auditd-conf.d-2.1.3 auditd
 
+	# Flatcar: install sample configuration
+	insinto /usr/share/auditd
+	doins "${FILESDIR}"/init.d/auditd.conf
+	newtmpfiles auditd.conf
```

This here is about the tmpfile .conf, so
```suggestion
	newtmpfiles "${FILESDIR}/auditd-conf.tmpfiles" auditd-conf.conf
```
and a new file `auditd-conf.tmpfiles` would contain `L   /etc/audit/auditd.conf   -       -       -       -       /usr/share/auditd/auditd.conf`